manage.py

```diff
@@ -241,6 +241,8 @@ def update_schemas(processes=1):
         path = os.path.join('data', 'sql_updates', '{}.sql'.format(task))
         execute_sql_file(path)
     execute_sql_file('data/rename_temporary_views.sql')
+    execute_sql_file('data/converted_mvs/sched_a_by_size_merged.sql')
+    execute_sql_file('data/converted_mvs/sched_a_by_state_recipient_totals.sql')
```

Comment:
Nice that these are in `update_schemas`, that makes it easier to ensure they're run appropriately!